spec/support/queries/filters/shared_filter_examples.rb

```diff
@@ -35,7 +35,13 @@
     described_class.create!(name: instance_key, context:, operator:, values:)
   end
   let(:name) { model.human_attribute_name((instance_key || expected_class_key).to_s.gsub("_id", "")) }
-  let(:model) { WorkPackage }
+  let(:model) do
+    if /^Queries::(?<model_plural>(?:[A-Z][a-z]+)+)::Filters?::(?:[A-Z][a-z]+)+Filter$/ =~ described_class.name
+      model_plural.singularize.constantize
+    else
+      raise "needs to be defined"
+    end
```

Comment:
Neat!

Reply:
Personally, I think we should get rid of ✨ magic 🪄 like this and have things like this be explicitly defined so that we don't have to guess what model is used but it is explicitly stated in the test. But that's not an issue of this PR and should be followed up in a Code Maintenance ticket.